scripts/repl.py

```diff
@@ -0,0 +1,66 @@
+"""This header drops you into a galaxy repl when run from $GALAXY_ROOT with -i"""
+import sys
+import os
+
+new_path = [ os.path.join( os.getcwd(), "lib" ) ]
+new_path.extend( sys.path[1:] )  # remove scripts/ from the path
+sys.path = new_path
+
+import logging
+logging.basicConfig()
+log = logging.getLogger(name=__name__)
+import galaxy.config
+from galaxy.objectstore import build_object_store_from_config
+from ConfigParser import ConfigParser
+import galaxy.model
+
+
+def dbinit(config_path):
+    config_parser = ConfigParser({
+        'here': os.getcwd(),
+        'database_connection': 'sqlite:///database/universe.sqlite?isolation_level=IMMEDIATE'
```

Comment:
Would it make sense to use the database configuration from `galaxy.ini` here? Maybe as option?